specs/protocol/derivation.md

```diff
@@ -535,7 +535,7 @@ describe the encoding of this polynomial.
 The field elements are encoded as big-endian integers (`KZG_ENDIANNESS = big`).
 
 To save computational overhead, only `254` bits per field element are used for rollup data.
-
+To optimize data encoding, `254` bits are defined as equivalent to `31.5` bytes. To ensure efficient and complete utilization of data, four elements are employed, summing to a total of `31.5` bytes multiplied by `4`, which yields `127` bytes.
```

Comment:
254 bits is equivalent to 31.75 bytes, not 31.5 bytes. Also please change the wording to be less verbose.

Reply:
LOL sorry for the stupid math problem. Shame on that, but good to know you got my point ;)